hardware/Utils/logger.py

```diff
@@ -32,11 +32,12 @@ def __init__(
         # show logs on screen
         self.showLogsOnScreen = show_logs or os.environ.get("SHOW_LOGS")
 
-    def get_logger_file(self, file_name):
+    def get_logger_file(self, filename):
         d = os.environ["LOG_DIRECTORY"]
+        filename += "-" + str(time.strftime("%Y%m%d-%H%M%S")) + ".log"
```

Comment:
this is a nit, but if we're going to use this format maybe we should replace the `-` with a `T` so that it conforms to ISO standards